test/routes/tasklist.test.js

```diff
@@ -76,24 +77,76 @@ afterEach(() => {
 })
 
 describe('GET /tasklist/', () => {
-  test('should render tasklist', () =>
-    request(app)
+  test('should render a tasklist for male prison', () => {
+    userService.getUser.mockResolvedValue({
```

Comment:
It still works without this mock where 'female: false'. Same down in lines 133, 205, 228. Is this mock unnecessary?